site/content/kapp-controller/docs/latest/faq.md

```diff
@@ -38,3 +38,59 @@ Even though the error above is reported, the Package will still be installed and
 should work as expected. It can also still be uninstalled by deleting the
 PackageInstall. The PackageRepository can be recreated and the PackageInstall
 will sync and reconcile without any updates needed to resolve the error.
+
+### How can I generate the valuesSchema from my ytt schema?
+
+If you are using `ytt` as your Package's templating option and have [defined a schema](../../../ytt/docs/latest/how-to-write-schema), you can use `ytt` to generate your `valuesSchema` (which is in OpenAPI v3 format) for you.
+
+This is the recommended workflow:
+
+1. Create an OpenAPI Document from a Data Values Schema file:
+
+    ```bash
+    $ ytt -f schema.yml --data-values-schema-inspect -o openapi-v3 >schema-openapi.yml
+    ```
+   
+    which will produce...
+
+    ```yaml
+    #! schema-openapi.yml
+    openapi: 3.0.0
+    info:
+      version: 1.0.0
+      title: Openapi schema generated from ytt schema
+    paths: {}
+    components:
+      schemas:
+        dataValues:
+          type: object
+          properties:
+            namespace:
+              type: string
+              default: fluent-bit
+    ```
+
+2. Turn your Package CR into a `ytt` template, itself:
```

Comment:
really nit-picking here but what does "itself" mean ?   or what meaning does the word "itself" add to this sentence?  I'm mostly asking because i'm wondering how to turn a package CR into a ytt template and what that would even mean. it's a package CR.  why would it want to be a ytt template?      is it possible to motivate -- i mean, i realize i'm on "step 2 of 3" but I'm looking for the "so that" -  like  "turn the pumpkin into a carriage so that Cinderella can ride it to the ball."   Otherwise we're just out here turning perfectly good pumpkins into ytt templates and i'm not sure why yet :)

Reply:
> ... Otherwise we're just out here turning perfectly good pumpkins into ytt templates ...

You say that as if it could be a bad thing... 😛 

Absolutely! I'll add, "... so that we can insert the schema definition in the right spot, automatically.

Reply:
done.